contributors/devel/release/patch_release.md

```diff
@@ -0,0 +1,78 @@
+# Building Kubernetes patch releases
+
+This document describes the process for creating Kubernetes patch releases and communicating publishing
+status regarding known issues impacting the current release.
+
+## TL;DR
+
+The patch release czar will summarize all issues discovered since the last patch release weekly
+at the SIG release meeting and publish the summary to kubernetes-sig-release@googlegroups.com.
+
+## Process
+
+After a Kubernetes minor release, patch releases are managed by the Kubenretes patch release czar.
+The patch release czar works with SIGs to coordinate merging cherrypicks into the release branch,
+then cutting and publishing a new patch release.
+
+## Communicating critical fixes to the community
+
+In order to provide better transparency with the community on the patch release status and process,
+the patch release czar provides a weekly summary of all issues that have been discovered since the
+*last patch release* (for the most recent minor release only).
+
+The summary will be presented at the weekly sig-release meeting as the first agenda item (recorded
+in the release notes), and then mailed to kubernetes-sig-release@googlegroups.com.  During / after
+the summary at the SIG meeting, the patch release czar can answer questions from the community.
+
+The email update includes the state and a short summary for each item using the following template.
```

Comment:
I'm not sure if it's what you meant by "auto-tagging", but I meant that we would define a set of tags that we expect humans to apply, like `status/under-investigation`. It's the same amount of human work as putting it into the template, except that the GitHub issue remains the single source of truth and the info is searchable. If there is no issue or the release note is confusing, I would rather correct those things in GitHub's database than write down a better copy in a static file.